### ISSUE TITLE
Candidate listing in CLI

### ISSUE THREAD

**Issue by roman-khimov** [nspcc-dev/neo-go#2067]
We have an RPC interface for candidate listing, but it's not convenient for regular people. neo-cli has `get candidates` command that outputs this list into more user-friendly fashion, so we need something like that. But I'd like it to output something like
```
key   votes  CN Committee
key1    XXX   Y Y
key2    XXX   Y Y
...
key8    XXX   N Y
...
key21   XXX   N N
```
Because CN/committee status is important.

**Comment by roman-khimov** [comment]
Fix document added by #2068 to use this command for status queries.

**Comment by roman-khimov** [comment]
See #2069 also, this might be `query candidates`. I think `query` can then be extended with more getters.
